src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
This block has margin: 0 by default, remove this styles

Reply:
Right! Thank you